mxcube3/routes/qutils.py

```diff
@@ -286,6 +286,11 @@ def _handle_dc(sample_id, node):
     parameters.pop('centred_position')
     queueID = node._node_id
     enabled, state = get_node_state(queueID)
+    try:
+        lims_data = mxcube.rest_lims.get_dc(node.id)
```

Comment:
@marcus-oscarsson I think the mockup lims hwobj should be of the 'ISPyBRest' type (correct me if I am wrong). This is why it is `mxcube.rest_lims`. So the `try...except` should not be needed, provided the underlying objet is the right one. So what are the steps to take ? Adding a new `ISPyBRest` mockup ? Fixing the current mockup to have a `get_dc()` ?